share/namespace.go

```diff
@@ -147,6 +150,9 @@ func (n Namespace) ValidateForData() error {
 // an error is returned indicating the issue. This ensures that only valid namespaces are
 // used when dealing with blob data.
 func (n Namespace) ValidateForBlob() error {
+	if err := n.validate(); err != nil {
+		return err
+	}
```

Comment:
We call ValidateForData below which calls `validate` already, so this call is redundant

Reply:
Good point. Will remove it

Reply:
https://github.com/celestiaorg/go-square/pull/145/commits/c870fc1aa826976e97c30ed50470f118f7fdaa67